reporting/reports.py

```diff
@@ -81,7 +82,7 @@ def _set(self, tag, value):
                         self.column_indexes = {}
                         self.column_types = {}
                         self.column_descriptions = {}
-                        lines = filter(lambda x: x != '', re.split('\s*\n\s*', value))
+                        lines = [x for x in re.split('\s*\n\s*', value) if x != '']
```

Comment:
Leave the `filter`, replace `lambda` with `None` which will simply throw away all elements where their `bool(value)` equals `False`.

Reply:
See my previous comment

Reply:
You too. :-)